src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo | null,
```

Comment:
You've already checked if selectedTodo exists at App.tsx before showing TodoModal, why do you need to consider this possibility here?
```suggestion
  todo: Todo,
```